src/main/kotlin/org/eln2/libelectric/sim/electrical/mna/impl/MnaCircuitImpl.kt

```diff
@@ -0,0 +1,103 @@
+package org.eln2.libelectric.sim.electrical.mna.impl
+
+import org.eln2.libelectric.sim.electrical.generic.*
+import org.eln2.libelectric.sim.electrical.mna.Circuit
+import org.eln2.libelectric.sim.electrical.mna.component.*
+
+class MnaCircuitImpl() : GenericCircuit {
+    var circuit = Circuit()
+    var components = mutableListOf<GenericComponent>()
+    var connectivity = mutableListOf<ComponentConnectivityMapping>()
+    override val backingImplementation = "ModifiedNodalAnalysis"
```

Comment:
Might not be the best description of this implementation: MNA is merely the algorithm. The implementation itself is a "pure Kotlin", "Apache Commons", etc. _implementation_ of MNA.

Reply:
Fair enough. This isn't really supposed to be an entirely descriptive identifier so much as a unique identifier; but your claim is solid. Should I change it to `native-kotlin-mna`?

Reply:
How about `kotlin-apache-mna`? The underlying math library might be useful information. ~~I'm definitely not working on `c-lapack-mna` c:~~